src/Form/FormAnalyzer.js

```diff
@@ -49,20 +56,37 @@ class FormAnalyzer {
         return this
     }
 
+    /**
+     *
+     * @param {object} p
+     * @param {'positive'|'negative' | ''} [p.matches] - Skip regexes and just update the value
```

Comment:
Do we have a description for what `matches` is, rather than the effect it'll have in the code below?
Something like your comment below looks good for this -- e.g.
```suggestion
     * @param {'positive'|'negative' | ''} [p.matches] - Explicit signal match direction when the signal is not directly related to `string`
```